### PR TITLE
01 training task

### PR DESCRIPTION
**Pull request by cherevachy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,13 @@
...
-The instructions on how to setup and generate tests project can be found in the official ["Getting Started"](https://docs.vividus.dev/vividus/latest/getting-started.html) guide.
+1.	Clone this repo to your local machine & build the project as described in the Module 4
 
-## Still have questions?
-:postbox: [Contact Us](https://docs.vividus.dev/vividus/latest/index.html#_contract_us)
+2.	Import the project to your Eclipse workspace:
+    - Click File > Import > Existing Projects into Workspace
+    - Set the path to the built project and click "Finish" button
+
+3.	Run the tests located in /demo folder at your local machine using the default config
```
Comment (uarlouski) [inline README.md:11]: how to run?

File: src/main/resources/properties/configuration.properties [hunk 2]
```diff
@@ -1,3 +1,13 @@
-configuration.profiles=
-configuration.suites=
-configuration.environments=
+# This config file is set up to -
+
+# execute scripts on your local machine in Chrome browser in desktop mode and default resolution:
+
+configuration.profiles=web/desktop/chrome
+
+# pick up suite related properties from 'demo' folder:
```
Comment (uarlouski) [inline src/main/resources/properties/configuration.properties:7]: from 'demo' suite folder?

File: src/main/resources/properties/configuration.properties [hunk 3]
```diff
@@ -1,3 +1,13 @@
...
+
+configuration.suites=demo
+
+# pick up environment related properties from 'saucedemo' folder:
```
Comment (uarlouski) [inline src/main/resources/properties/configuration.properties:11]: from 'saucedemo' environment folder?

File: build.gradle [hunk 4]
```diff
@@ -1,5 +1,5 @@
-project.description = 'put project description here'
-group = 'put your project group here, e.g. org.vividus.starter'
+project.description = 'VIVIDUS Training project'
+group = 'org.vividus.starter'
```
Comment (valfirst) [inline build.gradle:2]:
```suggestion
group = 'org.vividus.training'
```

File: README.md (hunk 1 above)
Comment (cherevachy) [reply README.md]: Updated to "Execute tests located in /demo folder at your local machine as described in the Module 5 using the default config"

File: src/main/resources/properties/configuration.properties (hunk 3 above)
Comment (cherevachy) [reply src/main/resources/properties/configuration.properties:11]: updated, thanks

**Review by ikalinin1 — APPROVED** [review]

File: README.md [hunk 5]
```diff
@@ -1,8 +1,13 @@
...
 
-## Still have questions?
-:postbox: [Contact Us](https://docs.vividus.dev/vividus/latest/index.html#_contract_us)
+2.	Import the project to your Eclipse workspace:
+    - Click File > Import > Existing Projects into Workspace
+    - Set the path to the built project and click "Finish" button
+
+3.	Execute tests located in /demo folder at your local machine as described in the Module 5 using the default config
+
+4.	Zip the folder with the resulting Allure Report and share it with your mentor
```
Comment (ikalinin1) [inline README.md:13]: Maybe trainee should share screenshots so we can make sure they are aware of how to open and review the report

Comment (cherevachy) [reply README.md:13]: Info on how to open & review the report is a part of module 5, starting from this point, they'll have to send their Allure reports to the mentor after each task, so I'm not sure if it makes sense to closely control this part..

**Pull request merged**